src/ray/raylet/scheduling/cluster_resource_scheduler.cc

```diff
@@ -102,6 +102,9 @@ void ClusterResourceScheduler::AddOrUpdateNode(
 
 void ClusterResourceScheduler::AddOrUpdateNode(int64_t node_id,
                                                const NodeResources &node_resources) {
+  RAY_LOG(DEBUG) << "AddOrUpdateNode "
+                 << NodeID::FromBinary(string_to_int_map_.Get(node_id)) << " "
```

Comment:
Isn’t this making logs really verbose? Since it is updated every 100ms.

Reply:
Since it's DEBUG level, it's fine? I find it's useful because the resource report can affect the following scheduling decisions.

Reply:
I found many verbose debug logs bother debugging when I am debugging other issues. It is probably okay to merge if you need it now (i can remove it later once we have a better way to observe this)

Reply:
Agree 100ms is a bit much even for DEBUG.

Reply:
ok, removed it.